test/e2e/proxy_test.go

```diff
@@ -150,16 +180,100 @@ func TestProxyFlow(t *testing.T) {
+				require.NoError(t, err)
+
+				// Find application and check, if it is updated
+				updatedApp := user.getApplication(t, proxyCl, applicationName)
+				assert.Equal(t, updatedApp.Spec.DisplayName, proxyApp.Spec.DisplayName)
+
+				// Check that the Application is updated using a regular client (non-proxy)
+				originalApp := user.getApplicationWithoutProxy(t, applicationName)
+				assert.Contains(t, originalApp.Spec.DisplayName, "updated application")
+			})
+
+			t.Run("use proxy to list a HAS Application CR in the user appstudio namespace", func(t *testing.T) {
+				// Get List of applications.
+				err := proxyCl.List(context.TODO(), applicationList, &client.ListOptions{Namespace: tenantNsName(user.compliantUsername)})
+				// User should be able to list applications
+				require.NoError(t, err)
+				assert.NotEmpty(t, applicationList)
+
+				// Check that the applicationList using a regular client (non-proxy)
+				applicationListWS := &appstudiov1.ApplicationList{}
+				err = user.expectedMemberCluster.Client.List(context.TODO(), applicationListWS, &client.ListOptions{Namespace: tenantNsName(user.compliantUsername)})
+				require.NoError(t, err)
+				require.NotEmpty(t, applicationListWS)
+				assert.Equal(t, applicationListWS.Items, applicationList.Items)
+			})
+
+			t.Run("use proxy to patch a HAS Application CR in the user appstudio namespace via proxy API", func(t *testing.T) {
+				// Patch application
+				applicationName := user.getApplicationName(1)
+				patchString := "Patched application for proxy test"
+				// Get application
+				proxyApp := user.getApplication(t, proxyCl, applicationName)
+				// Patch for DisplayName
+				patchPayload := []patchStringValue{{
+					Op:    "replace",
+					Path:  "/spec/displayName",
+					Value: patchString,
+				}}
+				patchPayloadBytes, _ := json.Marshal(patchPayload)
+
+				// Appply Patch
+				err := proxyCl.Patch(context.TODO(), proxyApp, client.RawPatch(types.JSONPatchType, patchPayloadBytes))
+				require.NoError(t, err)
+
+				// Get patched app and verify patched DisplayName
+				patchedApp := user.getApplication(t, proxyCl, applicationName)
+				assert.Equal(t, patchedApp.Spec.DisplayName, patchString)
+
+				// Double check that the Application is patched using a regular client (non-proxy)
+				wsApp := user.getApplicationWithoutProxy(t, applicationName)
+				assert.Equal(t, wsApp.Spec.DisplayName, patchString)
+			})
+
+			t.Run("use proxy to delete a HAS Application CR in the user appstudio namespace via proxy API and use websocket to watch it deleted", func(t *testing.T) {
+				// Delete applications
+				for i := 0; i < len(applicationList.Items); i++ {
+					// Get application
+					proxyApp := applicationList.Items[i].DeepCopy()
+					// Delete
+					err := proxyCl.Delete(context.TODO(), proxyApp)
+					require.NoError(t, err)
+					err = w.WaitForApplicationDeletion(
+						user.expectedMemberCluster.RetryInterval,
+						user.expectedMemberCluster.Timeout,
```

Comment:
I know that this was already there before, but we could use the "default constants" for these values inside of the `WaitFor*` methods instead of passing it every time.
https://github.com/codeready-toolchain/toolchain-e2e/blob/2b54e59598f40d93c94c2a457d5ebe96cca4f962/testsupport/wait/awaitility.go#L38-L39

Reply:
Resolved. Thank you!